src/app/test/helpers/general.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiErrorDetails } from "@baw-api/api.interceptor.service";
+import { Subject } from "rxjs";
+
+/**
+ * Default step size for timeouts
+ */
+export const testStepInterval = 16;
```

Comment:
Could this be 0?

If you're wondering how to do n steps (where n > 0) then you do multiple `setTimeouts`, one for each `steps`

Reply:
If my suggestion (or assertion does not work, then leave a comment explaining why you settled on 16)

Reply:
So each step triggers the next step recursively?

Ie.

```typescript
setTimeout(() => {
  // step 0
  setTimeout(() => {
    // step 1
  }. 0);
}, 0);
```

Reply:
yeah recursive, though more like:

```
var waitOne = setTimout(() => {
  if (stepsRemaining >0) {
    stepsRemaining--;
   waitOne(); 
  }
}, nextRender);
```